commands/gitService.go

```diff
@@ -112,7 +113,11 @@ func (g *GS) UpdateGitViaGit(c *DeployCmd, response UploadResponse) error {
 	if err != nil {
 		return err
 	}
-	commitHash, err := w.Commit("[sectionctl] updated nodejs/.section-external-source.json with new deployment.", &git.CommitOptions{})
+	commitHash, err := w.Commit("[sectionctl] updated nodejs/.section-external-source.json with new deployment.", &git.CommitOptions{Author: &object.Signature{
+		Name:  "sectionctl",
+		Email: "noreply@section.io",
```

Comment:
Can we obtain their actual name and email address from their API token using an extra API call?
e.g. https://aperture.section.io/api/ui/#!/User/currentUserGet

Reply:
oh we definitely could.